### PR TITLE
test: front-end e2e test for rawDataEditor

### LINKED ISSUE #1506: test: front-end e2e test for rawDataEditor
# Feature request

## Please describe your feature
Need front-end e2e test for rawDataEditor.
- [ ] route
- [ ] consumer
- [ ] service
- [ ] upstream

refer pr: #1505

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
resolve #1506 
___
### Please add the corresponding test cases if necessary.
Added tests for Route, Service, Upstream, Consumer for rawDataEditor
___

### CODE REVIEW

File: web/cypress/integration/consumer/create_and_delete_consumer.spec.js [hunk 1]
```diff
@@ -56,6 +56,21 @@ context('Create and Delete Consumer', () => {
...
   });
 
+  it('should view the consumer', function () {
+    cy.visit('/');
+    cy.contains('Consumer').click();
+
+    cy.get(this.domSelector.nameSelector).type(this.data.consumerName);
+    cy.contains('Search').click();
+    cy.contains(this.data.consumerName).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/consumer/create_and_delete_consumer.spec.js:66]:
```suggestion
    cy.get(this.domSelector.xxx).should('be.visible');
```

File: web/cypress/integration/route/create-edit-delete-route.spec.js [hunk 2]
```diff
@@ -85,6 +85,21 @@ context('Create and Delete Route', () => {
...
   });
 
+  it('should view the route', function () {
+    cy.visit('/');
+    cy.contains('Route').click();
+
+    cy.get(this.domSelector.nameSelector).type(name);
+    cy.contains('Search').click();
+    cy.contains(name).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/route/create-edit-delete-route.spec.js:95]: ditto

File: web/cypress/integration/route/create-edit-delete-route.spec.js [hunk 3]
```diff
@@ -103,6 +118,15 @@ context('Create and Delete Route', () => {
     cy.contains('Goto List').click();
     cy.url().should('contains', 'routes/list');
     cy.contains(newName).siblings().should('contain', this.data.description2);
+
+    // test view
+    cy.contains(newName).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/route/create-edit-delete-route.spec.js:124]: ditto

File: web/cypress/integration/service/create-edit-delete-service.spec.js [hunk 4]
```diff
@@ -40,6 +40,21 @@ context('create and delete service ', () => {
...
   });
 
+  it('should view the service', function () {
+    cy.visit('/');
+    cy.contains('Service').click();
+
+    cy.get(this.domSelector.nameSelector).type(this.data.serviceName);
+    cy.contains('Search').click();
+    cy.contains(this.data.serviceName).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/service/create-edit-delete-service.spec.js:50]: ditto

File: web/cypress/integration/service/create-edit-delete-service.spec.js [hunk 5]
```diff
@@ -59,10 +74,20 @@ context('create and delete service ', () => {
     cy.contains('Next').click();
     cy.contains('Submit').click();
     cy.get(this.domSelector.notification).should('contain', this.data.editServiceSuccess);
+
+    // test view
+    cy.contains(this.data.serviceName2).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/service/create-edit-delete-service.spec.js:80]: ditto

File: web/cypress/integration/upstream/create_and_delete_upstream.spec.js [hunk 6]
```diff
@@ -41,6 +41,22 @@ context('Create and Delete Upstream', () => {
...
   });
 
+  it('should view the (roundrobin) upstream', function () {
+    cy.visit('/');
+    cy.contains('Upstream').click();
+
+    cy.get(this.domSelector.nameSelector).type(this.data.upstreamName);
+    cy.contains('Search').click();
+    cy.contains(this.data.upstreamName).siblings().contains('View').click();
+    cy.get('.ant-drawer-content').should('be.visible');
```
Comment (LiteSun) [inline web/cypress/integration/upstream/create_and_delete_upstream.spec.js:51]: ditto

File: web/cypress/integration/service/create-edit-delete-service.spec.js [hunk 7]
```diff
@@ -59,10 +74,20 @@ context('create and delete service ', () => {
...
+
+    cy.get('.CodeMirror-scroll').within(() => {
```
Comment (LiteSun) [inline web/cypress/integration/service/create-edit-delete-service.spec.js:82]:
```suggestion
    cy.get(this.domSelector.xxx).within(() => {
```

**Review by LiteSun — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Pull request merged**